dataland-frontend/src/utils/SessionTimeoutUtils.ts

```diff
@@ -17,21 +17,19 @@ const minRequiredRemainingValidityTimeOfRefreshTokenDuringCheck = TIME_DISTANCE_
  * @param forceStoringValues forces storing the refresh token and its expiry timestamp in the shared storage, even if
  * the updateToken() has not done an update itself because of the minValidity value
  */
-export function updateTokenAndItsExpiryTimestampAndStoreBoth(keycloak: Keycloak, forceStoringValues = false): void {
-  keycloak
-    .updateToken(60)
```

Comment:
Is this no longer necessary?

Reply:
Correct. It is done automatically now on all requests that require it.

Reply:
Ok